cmd/helm/reset.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"k8s.io/helm/cmd/helm/helmpath"
+	"k8s.io/helm/cmd/helm/installer"
+	"k8s.io/helm/pkg/helm"
+	"k8s.io/helm/pkg/kube"
+	"k8s.io/helm/pkg/proto/hapi/release"
+)
+
+const resetDesc = `
+This command uninstalls Tiller (the helm server side component) from your
+Kubernetes Cluster and optionally deletes local configuration in
+$HELM_HOME (default ~/.helm/)
+`
+
+type resetCmd struct {
+	force          bool
+	removeHelmHome bool
+	namespace      string
+	out            io.Writer
+	home           helmpath.Home
+	client         helm.Interface
+	kubeClient     internalclientset.Interface
+	kubeCmd        *kube.Client
+}
+
+func newResetCmd(client helm.Interface, out io.Writer) *cobra.Command {
+	d := &resetCmd{
+		out:    out,
+		client: client,
+	}
+
+	cmd := &cobra.Command{
+		Use:               "reset",
+		Short:             "destroys Helm",
```

Comment:
This should probably be a bit more descriptive. Like "uninstalls Tiller from a cluster"